loadtest/user/userentity/actions.go

```diff
@@ -1515,3 +1591,40 @@ func (ue *UserEntity) prepareRequest(method, url string, data io.Reader, headers
 
 	return rq, nil
 }
+
+func (ue *UserEntity) getGqlResponse(input any) (*graphql.Response, error) {
+	var gqlResp *graphql.Response
+	buf, err := json.Marshal(input)
+	if err != nil {
+		return nil, err
+	}
+
+	req, err := ue.prepareRequest(http.MethodPost,
+		getGQLURL(ue.client.URL),
+		bytes.NewReader(buf),
+		map[string]string{})
+	if err != nil {
+		return nil, err
+	}
+
+	resp, err := ue.client.HTTPClient.Do(req)
+	if err != nil {
+		return nil, err
+	}
+	defer closeBody(resp)
+
+	err = json.NewDecoder(resp.Body).Decode(&gqlResp)
```

Comment:
Super nit: can we move the definition of `gqlResp` right before its first usage, which happens in this line?

Reply:
Done